salt/modules/zypper.py

```diff
@@ -747,11 +747,12 @@ def mod_repo(repo, **kwargs):
         Enable or disable (True or False) GPG check for this repository.
 
     gpgautoimport
-        Automatically trust and import new repository.
+        If set to True, automatically trust and import public GPG key for
```

Comment:
This seems to imply that `gpgautimport` defaults to `False`, but maybe we should be explicit about that information and say directly what the default is.